### ISSUE TITLE
broadcast support?

### ISSUE THREAD

**Issue by ahnitz** [adrn/schwimmbad#15]
I am using schwimmbad with some extensions for our particular work load. Specifically, we periodically need to guarantee that we run a function on each member of the pool. Would that be something that would be within the scope of this library, or is it too specialized? If it's within scope, I'd be interested in contributing the code here.

**Comment by dfm** [comment]
I'm not the boss (@adrn), but I don't think that this is out of scope and I think it would be a useful addition. I don't normally use the `bcast` feature that exists in `emcee.MPIPool`, but I know that others do. If you put this together, could you also write up an example usage? Thanks!

**Comment by adrn** [comment]
Agreed! I would definitely welcome that, but just ask for some examples / documentation.

Thanks!
